merlin/datasets/ecommerce/__init__.py

```diff
@@ -20,5 +20,12 @@
     prepare_alliccp,
     transform_aliccp,
 )
+from merlin.datasets.ecommerce.dressipi.dataset import get_drepessi2022
 
-__all__ = ["prepare_alliccp", "transform_aliccp", "get_aliccp", "default_aliccp_transformation"]
+__all__ = [
+    "prepare_alliccp",
+    "transform_aliccp",
+    "get_aliccp",
+    "default_aliccp_transformation",
+    "get_drepessi2022",
```

Comment:
`get_drepessi2022` there is a typo in this func name.

Reply:
Fixed the typo.